sphinx_gallery/notebook.py

```diff
@@ -64,14 +68,19 @@ def directive_fun(match, directive):
                     match.group(1).strip()))
 
 
-def rst2md(text, heading_levels):
-    """Converts the RST text from the examples docstrings and comments
+def rst2md(text, gallery_conf, target_dir, heading_levels):
+    """Converts the RST text from the examples docstrigs and comments
```

Comment:
Hmm how did we go back to 'docstrigs'? Bad merge?

Reply:
Bad conflict resolution on rebase. Fixed now.